### PR TITLE
kv subject prefix changes to properly support accounts

### PR DESCRIPTION
**Pull request by scottf**

### CODE REVIEW

File: src/NATS.Client/Internals/Validator.cs [hunk 1]
```diff
@@ -543,5 +543,21 @@ public static Duration EnsureDurationNotLessThanMin(long providedMillis, Duratio
         {
             return EnsureNotNullAndNotLessThanMin(Duration.OfMillis(providedMillis), minimum, dflt);
         }
+         
+        public static bool Equal(byte[] a, byte[] a2) {
```
Comment (sixlettervariables) [inline src/NATS.Client/Internals/Validator.cs:547]: `a.SequenceEqual(b)` would be preferred.

Comment (scottf) [reply src/NATS.Client/Internals/Validator.cs:547]: Either or both a and a2 can be null. SequenceEqual throws on nulls.

Comment (sixlettervariables) [reply src/NATS.Client/Internals/Validator.cs:547]: Past the null checks I meant, was reviewing on my phone.

Comment (scottf) [reply src/NATS.Client/Internals/Validator.cs:547]: I initially considered and looked at SequenceEqual code. Its generic, it instantiates a comparer object, checks for nulls to throw, checks for a known collection type, so much just to compare a simple byte array? This code is pretty simple, I think it's ok, but if it's absolutely have to be done I will change it.

**Review by aricart — APPROVED** [review]
LGTM

**Pull request merged**